models/maat/R/model2netcdf.MAAT.R

```diff
@@ -25,9 +25,6 @@ model2netcdf.MAAT <- function(outdir, sitelat = -999, sitelon = -999, start_date
   ## TODO is it OK to give site lat/long -999 if not running at a 'site'? 
   ## TODO !!UPDATE SO IT WILL WORK WITH NO MET AND WITH MET DRIVER!!
   
-  ### Load required libraries
-  library(PEcAn.utils)
```

Comment:
here you've dropped PEcAn.utils but haven't added any pkg::fcn refs -- was utils just not used?